include/interface/commandStructure.h

```diff
@@ -48,12 +48,12 @@ typedef struct __attribute__((__packed__)) _commandStatusStructure {
 } commandStatusStructure_t;
 
 typedef enum {
-    // Controller internal use
-    Commander_Self = 1,
     // Graphical user interface
-    Commander_GUI = 2,
+    Commander_GUI = 1,
     // Commandable SAL component
-    Commander_CSC = 3,
+    Commander_CSC = 2,
+    // Controller internal use
+    Commander_Self = 3,
```

Comment:
Please expand the code comment to explain what it is for. I did a global search in the code and found no instances. Naively I would expect "internal use" to mean "within this package" but unless I missed something that is not what it means (at least not yet).

Reply:
Improve the document in commit: 8045b96. Thanks!